tests/osaka/eip7951_p256verify_precompiles/test_eip_mainnet.py

```diff
@@ -0,0 +1,91 @@
+"""
+Tests for [EIP-7951: Precompile for secp256r1 Curve Support](https://eips.ethereum.org/EIPS/eip-7951).
+"""
+
+import pytest
+from execution_testing import (
+    Alloc,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+
+from .spec import H, R, S, Spec, X, Y, ref_spec_7951
+
+REFERENCE_SPEC_GIT_PATH = ref_spec_7951.git_path
+REFERENCE_SPEC_VERSION = ref_spec_7951.version
+
+
```

Comment:
If we add this here we can remove Osaka mark's below.
```suggestion
pytestmark = [pytest.mark.valid_at("Osaka"), pytest.mark.mainnet]
```